packages/utils-helpers/src/getVersion.js

```diff
@@ -0,0 +1,5 @@
+const getVersion = () => {
+    return 'v1.0.3';
```

Comment:
Instead of parsing this file again, let's just re-create it completely in the bash script :) Makes it more robust I'd say